CHANGELOG.md

```diff
@@ -1,6 +1,252 @@
 Change Log
 ==========
 
+# Version 3.8.3
+
+_2024-03-20_
```

Comment:
Do we need to update the changelog in the 3.x branch?

Reply:
What if we change a wording in `main`? We now have to backport? It's ok to backport for code but for changelog, I'd rather have only a single source of truth.

Reply:
Fair enough - I reverted